sources/api/shibaken/src/warmpool/test_config.toml

```diff
@@ -0,0 +1,2 @@
+should_wait = true
+marker_path = "/var/lib/bottlerocket/shibaken.ran"
```

Comment:
Why does this file need to be checked in if the unit tests are writing a temporary copy of these configs?